frontend/src/lib/i18n/locales/en.json

```
@@ -184,6 +184,18 @@ the [Linguistics Institute at Payap University](https://li.payap.ac.th/) in Chia
       "user_not_verified": "User has not verified their email address",
       "user_needs_to_relogin": "Added members will need to log out and back in again before they see the new project.",
     },
+    "bulk_add_members": {
+      "add_button": "Bulk Add Members",
+      "explanation": "Adds all the provided Send/Receive logins/usernames to this project. New users will be created automatically with the provided password for logins that don't exist.",
+      "modal_title": "Add many Members to this project",
+      "submit_button": "Bulk Add/Create Members",
+      "finish_button": "Finish Bulk Add/Create",
```

"Finish" suggests that this button might be necessary to complete the process, but it's actually already finished.
```suggestion
      "finish_button": "Close",
```

---

Done in commit e1391d6498280e77fd10d35730508154d351e23b.